BUILDER_DEV.md

```diff
@@ -293,3 +293,69 @@ Add the following line if not present to see if it resolves your issue:
    says `WARNING: out of shared memory`, edit the `postgresql.conf` file in
    `/hab/pkgs/core/postgresql/$VERSION/$RELEASE/config` and add
    `max_locks_per_transaction=128` to it.
+
+### Node troubleshooting
+
+1. If you are seeing an error similar to this:
+
+   ```
+      web.1       | /home/nell/habitat/support/builder_web.sh: line 38: ./node_modules/.bin/lite-server: No such file or directory
+   ```
+
+1. Then check whether you are having node version issues.  Get the version builder-web expects like this:
+
+   ```
+     cd components/builder-web
+     node -v
```

Comment:
It's worth noting that this will only tell you the version that `builder-web` expects if you already have nvm installed.  The reason why is because nvm will source the `.nvmrc` file in that directory and attempt to switch to that version of node automatically, similar to how rvm used to work.

The only way to really know what version `builder-web` expects is to `cat .nvmrc`.

If you run the above commands and you _don't_ have nvm installed, then you'll find out what version of node is globally installed on your system, not the version that `builder-web` expects.